src/main.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
 
     var opts = cli.opts;
 
-    var bumperConfig = merge( { }, cli.getConfig(), opts );
+    var bumperConfig = merge( { }, opts, cli.getConfig()  );
```

Comment:
Hi @malys, will accept the change, but this is not really the proper fix. 

So basically the idea was to make it easy to override any value passed in a config file, but since the options had several default values they were overriding the ones in the config. And that should only pass if a parameter was specified. 

I will keep open your bug, since the proper fix for that will be to remove all the `default` entries in the `options.js` file and move those defaults to the first parameter of this merge statement. That way it will work as it should.

But for now is good enough.